### DISCUSSION TITLE
posts/Go/golang-menu/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#367] [Announcements]
https://www.liwenzhou.com/posts/Go/go_menu/ 

 李文周的Blog

**Comment by meplay** [comment]
大佬，更新快点啊，，等着你，加油！！！

**Comment by printfaiz0710** [comment]
OMO

**Comment by jacobwwang** [comment]
持续关注，感谢大佬！

**Comment by siatzhuliang** [comment]
大佬讲的很棒，非常感谢

**Comment by c-jf** [comment]
大佬大佬卧槽，后面的也都出写视频啊

**Comment by adeljck** [comment]
麻烦大佬讲解一下连接mongodb数据库，感觉好多坑

**Comment by Q1mi** [comment]
> @meplay
> 大佬，更新快点啊，，等着你，加油！！！

好的，我看一下最近会更新上

**Comment by kokohui** [comment]
哦, 青蛙王子, 请收下的膝盖!

**Comment by Ronnri** [comment]
2019/11/10,打卡刷完Go基础知识，一切才刚刚开始~！ 谢谢大佬~~~ life is short, let's GO!

**Comment by JackDemo** [comment]
goGoGO

**Comment by Alleyoops** [comment]
老师您好，我是今年的大一新生，对您的视频很感兴趣，也很感谢，很满意，我们学校在学C语言，但我也爱上了go语言，觉得它很高级，奈何自学难度太大，细节一点的知识零零散散，已经想过放弃了，今天发现了您的B站视频，觉得放弃又心有不甘，希望老师的视频和博客可以长期保持更新，让您的博客能够成为一群golang追随者的宝地，让我们时刻有兴趣和动力去解决和接受golang每一个细节和难点。谢谢老师，我也想成为您这样的大佬！！！
如果必要的话，花钱也值得啊，我觉得老师真得讲得详细具体啊！

**Comment by Leo2019-ok** [comment]
我是一名萌新，学了一些函数、指针、遍历课程后，感觉越来越学不进去了，不理解这些在现实场景中是用来解决什么问题的，很希望能自己边学边写一个网站，这样就比较好理解这些词汇的意思

**Comment by gjwlinux** [comment]
大佬，更新快点啊，，等着你，加油！！

**Comment by Muromi-Rikka** [comment]
有没有打算讲一讲go micro

**Comment by TheGroundhogDay** [comment]
如果有 go micro和go web实战视频  就更好了。

**Comment by Q1mi** [comment]
> @Leo2019-ok
> 我是一名萌新，学了一些函数、指针、遍历课程后，感觉越来越学不进去了，不理解这些在现实场景中是用来解决什么问题的，很希望能自己边学边写一个网站，这样就比较好理解这些词汇的意思

B站有更新Go Web系列教程

**Comment by Q1mi** [comment]
> @Muromi-Rikka
> 有没有打算讲一讲go micro

目前在B站更新Go web系列，微服务在计划中。

**Comment by Q1mi** [comment]
> @TheGroundhogDay
> 如果有 go micro和go web实战视频  就更好了。


> @TheGroundhogDay
> 如果有 go micro和go web实战视频  就更好了。

目前在B站更新Go web系列，微服务在计划中。

**Comment by yzdzh** [comment]
讲得很好，受益匪浅，大佬有空的话可以把后面的也出下视频，能帮助很多初学者少走弯路

**Comment by zhencai1** [comment]
安装Go语言开发工具包，两种方法我都试了，还是显示加载失败，怎么解决求指教

**Comment by Q1mi** [comment]
> @zhencai1
> 安装Go语言开发工具包，两种方法我都试了，还是显示加载失败，怎么解决求指教
> 

周末更新了下百度网盘的文件，你再试下。不行就换Goland编辑器。

**Comment by zhencai1** [comment]
ok

**Comment by songzhibin97** [comment]
大佬我从入门就看你视频(Python -> Go) 膜拜

**Comment by LHgofors** [comment]
打卡第一天， 决定开始跟着大佬学GO

**Comment by QiuSoft** [comment]
沙河小王子，goLand好用的插件能介绍一下吗？

**Comment by VEH47** [comment]
帮忙发个B站视频的链接哦, 一直找不到. 之前看的第4期的, 好像跟第五期的还不太一样啊

**Comment by honkkki** [comment]
沙河小王子牛逼

**Comment by yusonghu** [comment]
# 人生苦短
`let's go`

**Comment by debuginn** [comment]
我也住在沙河  沙河666

**Comment by tomren** [comment]
感谢你的视频教程！

**Comment by russell662** [comment]
请问大佬的博客系统是自己写的吗?
毕设要用go写一个博客
如果可以参考一下设计什么的就好了

**Comment by debuginn** [comment]
> @YangLin1998
> 请问大佬的博客系统是自己写的吗?
> 毕设要用go写一个博客
> 如果可以参考一下设计什么的就好了

这个应该是
hexo+github的

**Comment by yzdzh** [comment]
提个小建议，这个博客网站加个搜索功能，方便其他人阅读其他文章

**Comment by erik-chen** [comment]
大佬，我也用hugo进行了部署，theme用maupassant。有个问题请教一下，我的主页上(https://erik-chen.github.io/)对文章会选取前面一部分显示，但很乱。怎么才能控制显示的部分呢？比如只显示一句话这样呢？

**Comment by Q1mi** [comment]
> @YangLin1998
> 请问大佬的博客系统是自己写的吗?
> 毕设要用go写一个博客
> 如果可以参考一下设计什么的就好了

我用的hugo

**Comment by Q1mi** [comment]
> @erik-chen
> 大佬，我也用hugo进行了部署，theme用maupassant。有个问题请教一下，我的主页上(https://erik-chen.github.io/)对文章会选取前面一部分显示，但很乱。怎么才能控制显示的部分呢？比如只显示一句话这样呢？

我现在是在文章里手动加 标识，前面是预览内容，后面是正文。

**Comment by Q1mi** [comment]
> @yizhidaozuihou
> 提个小建议，这个博客网站加个搜索功能，方便其他人阅读其他文章

好的，感谢建议。搜索功能一直有考虑做。

**Comment by Q1mi** [comment]
> @tomren
> 感谢你的视频教程！

感谢支持！

**Comment by momsenkung** [comment]
感谢大佬 教程很详细

**Comment by helloAYang** [comment]
在看，大佬的视频！

**Comment by helloAYang** [comment]
人生苦短，let's go

**Comment by Fanlen** [comment]
大佬看了你gin框架视频受益匪浅，期待出一些项目中常用的分页类 无限级分类 日志类等封装的系列，还有apidocs类似这样的。   也期待你出一些商业课程，从0开始搭建一个完整项目比如商城。我相信很多铁粉和我一样，会第一时间支持付费的。感谢你之前出的免费课程，受益类众多学习者

**Comment by lamubida** [comment]
人生苦短，let's go！第一天

**Comment by lamubida** [comment]
人生苦短，let's go！第二天

**Comment by iuuuuuaena** [comment]
# 谢谢大佬
## 谢谢大佬
### 谢谢大佬 
#### 谢谢大佬

**Comment by jinshiyuan1123** [comment]
刚来学习！顶起来！！

**Comment by openstack-test** [comment]
Golang学习手册之带你21周搞定Go语言视频怎么在b站上没有了呢，七米老师

**Comment by lamubida** [comment]
人生苦短，let's go！第三天

**Comment by Q1mi** [comment]
> @openstack-test
> Golang学习手册之带你21周搞定Go语言视频怎么在b站上没有了呢，七米老师

你看的应该是某些人上传的我在上家公司讲课的视频，被我上家公司在B站申明版权保护了。不是我上传的，不要再问我了。。。

**Comment by zys1317** [comment]
喂 同志&nbsp; 好几天没见你打卡了啊 继续啊 &nbsp;

 &nbsp;


也许我只属于这个虚拟的世界，因为这里看不清一个人，不要让我懂你，做虚拟的你，那样的天空才会蓝。。。


&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "notifications"<notifications@github.com&gt;; 
发送时间: 2020年4月21日(星期二) 上午10:48
收件人: "Q1mi/BlogComments"<BlogComments@noreply.github.com&gt;; 
抄送: "sevirce"<8548847@qq.com&gt;; "Comment"<comment@noreply.github.com&gt;; 
主题: Re: [Q1mi/BlogComments] Go语言学习之路 | 李文周的博客 (#3)





  
@openstack-test
 Golang学习手册之带你21周搞定Go语言视频怎么在b站上没有了呢，七米老师
  
你看的应该是某些人上传的我在上家公司讲课的视频，被我上家公司在B站申明版权保护了。不是我上传的，不要再问我了。。。

**Comment by jipyu** [comment]
> @lamubida
> 人生苦短，let's go！第二天

兄弟 还有后续嘛? 就Let's go了两天嘛?😁

**Comment by zys1317** [comment]
这是又开始了吗

 &nbsp;


也许我只属于这个虚拟的世界，因为这里看不清一个人，不要让我懂你，做虚拟的你，那样的天空才会蓝。。。


&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "notifications"<notifications@github.com&gt;; 
发送时间: 2020年5月18日(星期一) 下午5:34
收件人: "Q1mi/BlogComments"<BlogComments@noreply.github.com&gt;; 
抄送: "sevirce"<8548847@qq.com&gt;; "Comment"<comment@noreply.github.com&gt;; 
主题: Re: [Q1mi/BlogComments] Go语言学习之路 | 李文周的博客 (#3)





  
@lamubida
 人生苦短，let's go！第二天
  
兄弟 还有后续嘛? 就Let's go了两天嘛?😁

**Comment by bamdra** [comment]
非常棒啊！！！加QQ群拒绝新用户加入啊。。。

**Comment by Q1mi** [comment]
> @hfge
> 非常棒啊！！！加QQ群拒绝新用户加入啊。。。

QQ群设置了付费进群，请使用最新版手机QQ尝试加群。

**Comment by chlbupt** [comment]
发现了宝藏

**Comment by xdr630** [comment]
你好

**Comment by iTriumph** [comment]
大佬讲的很棒，非常感谢!

**Comment by catkukun** [comment]
这个github登录怎么实现的啊

**Comment by ghost** [comment]
太好了

**Comment by ghost** [comment]
终于找到了

**Comment by xjb714** [comment]
感谢楼主分享，加油学习

**Comment by siwenyuzhong** [comment]
请问大佬有B站配套那个视频的聊天室源码吗？那四个视频没有声音，对着敲了好几遍，都没运行成功，感谢🙏

**Comment by YanHeDoki** [comment]
强的

**Comment by zengmark** [comment]
大佬，这个界面太难整了吧，能换回之前那样吗，想要找知识点感觉没之前方便了

**Comment by zys1317** [comment]
为什么抄送给我好烦躁




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2025年6月14日(星期六) 中午11:40
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Q1mi/BlogComments] posts/Go/golang-menu/ (Discussion #367)














 
大佬，这个界面太难整了吧，能换回之前那样吗，想要找知识点感觉没之前方便了

**Comment by consolexinhun** [comment]
切片的链接填错了，填的还是数组的

**Comment by chenxingithub** [comment]
文章排版是换了风格了吗？这个导图的模式没有以前那么直接明了。

**Comment by consolexinhun** [comment]
我也建议换成之前的样子，现在的这个看起来太不方便了！！！

**Comment by GoldPuzzleCN** [comment]
太强了 直接跟着学
